go-controller/pkg/kubevirt/pod.go

```diff
@@ -683,3 +711,63 @@ func newRouterAdvertisementWithPrefixInfos(ip net.IP, destinationMAC net.Hardwar
 		PrefixInfos:    prefixInfos,
 	}
 }
+func findRunningPodsIPsFromPodSubnet(watchFactory *factory.WatchFactory, podAnnotation *util.PodAnnotation, nadName string) ([]*net.IPNet, error) {
+	nodeOwningSubnet, err := findNodeOwningSubnet(watchFactory, podAnnotation, nadName)
+	if err != nil {
+		return nil, err
+	}
+	if nodeOwningSubnet == nil {
+		return nil, nil
```

Comment:
Shouldn't this be an error?

Reply:
For example if we migrate a VM, and then that node goes down this will be the case and since we don't have nothing to GARP for we just return empty here.